keyboards/skyloong/dt40/info.json

```diff
@@ -0,0 +1,99 @@
+{
+    "manufacturer": "JZ-Skyloong",
+    "keyboard_name": "DT40",
+    "maintainer": "JZ-Skyloong",
+    "bootloader": "stm32duino",
+    "diode_direction": "ROW2COL",
+    "processor": "STM32F103",
+    "url": "https://github.com/JZ-Skyloong",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0040",
+        "vid": "0x1EA7"
```

Comment:
This vendor ID is already in use, and should be changed. 
https://www.the-sz.com/products/usbid/index.php?v=0x1ea7&p=&n=

Ideally, it should be above 0x4000, as that hasn't been assigned yet.  And the higher, the better.

Reply:
We are a mainland in china registered company of semitek and share the same VID. Our company has unified management of PID, which can ensure that ID is uniquely identified.